server/datastore/mysql/migrations/data/20181119180000_DeleteSoftDeletedEntities.go

```diff
@@ -1,54 +0,0 @@
-package data
```

Comment:
Deleting migrations that have been published in a release can lead to inconsistencies. We need to leave these migrations and assume they have been run prior to the new migration running.